src/init.cpp

```diff
@@ -438,8 +438,8 @@ void SetupServerArgs()
 #else
     hidden_args.emplace_back("-upnp");
 #endif
-    gArgs.AddArg("-whitebind=<addr>", "Bind to given address and whitelist peers connecting to it. Use [host]:port notation for IPv6", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
-    gArgs.AddArg("-whitelist=<IP address or network>", "Whitelist peers connecting from the given IP address (e.g. 1.2.3.4) or CIDR notated network (e.g. 1.2.3.0/24). Can be specified multiple times."
+    gArgs.AddArg("-whitebind=<[(permissions,)*@]addr>", "Bind to given address and whitelist peers connecting to it. Use [host]:port notation for IPv6.  Available permissions are: 'forcerelay,relay,noban,mempool,bloomfilter'. If the permission marker '@' is not specified, it will default to 'noban,mempool' with `whitelistrelay` and `whitelistforcerelay` optionally adding 'relay' and 'forcerelay' respectively. Can be specified multiple times.", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
+    gArgs.AddArg("-whitelist=<[(permissions,)*@]IP address or network>", "Whitelist peers connecting from the given IP address (e.g. 1.2.3.4) or CIDR notated network (e.g. 1.2.3.0/24). Available permissions are: 'forcerelay,relay,noban,mempool,bloomfilter'. If the permission marker '@' is not specified, it will default to 'noban,mempool' with `whitelistrelay` and `whitelistforcerelay` optionally adding 'relay' and 'forcerelay' respectively. Can be specified multiple times."
```

Comment:
`whitelistrelay` is true by default, whereas `whitelistforcerelay` defaults to false, so a better wording would be:

```
If the permission marker '@' is not specified, it will default to 'noban,mempool,whitelistrelay', unless `-whitelistrelay=0`. If `-whitelistforcerelay=1` then 'forcerelay' is also set by default.
```

Reply:
Would it be possible to factor out the string with the list of permissions to a single symbol (mabye in net_permissions.h?)
Can you split the cpp string literal into muliple lines?

```cpp
   "safd1. "
   "foo2. "
   "foobar3. (default...)"

Reply:
Will do after #16555 get merged.